index.js

```diff
@@ -0,0 +1,73 @@
+// Implement a `validateLineup` 
+// function that will return `true` if the lineup provided adheres to all of 
+// the rules listed below.
+
+// Lineup Rules
+// The total salary of all players in a lineup may not exceed $45,000
+// Lineups may not contain: 
+//                         more than 2 players from a single team
+//                         more than 3 players from a single game
+
+// Lineups must contain exactly 3 players with the position of 'OF' and must 
+// also contain exactly 1 player from each of the following positions: 
+
+// lineup.length : Need to loop through the lineup to get to the TEAM ID
+const validateLineup = (lineup) => {
+  allLineupsGood = true
+  lineup.forEach(lineup => {
+    intSalary = parseInt(lineup.salary)
```

Comment:
The only reason your tests passed is because you set your variables to true so they are false positives.  The rest of the code that you wrote does not get used at all throughout the rest of your project.